replication-api-impl/src/main/java/com/connexta/ion/replication/api/impl/ReplicatorRunner.java

```diff
@@ -67,8 +67,7 @@ public ReplicatorRunner(
   }
 
   public void init() {
-    scheduledExecutor.scheduleAtFixedRate(
-        this::scheduleReplication, STARTUP_DELAY, period, TimeUnit.SECONDS);
+    scheduledExecutor.scheduleAtFixedRate(this::scheduleReplication, 0, period, TimeUnit.SECONDS);
```

Comment:
I don't think we need a delay anymore. I think we're only waiting on solr to come up and spring-data-solr should retry any failed connection attempts.